packages/p2p/source/downloader/block-downloader.ts

```diff
@@ -134,9 +134,11 @@ export class BlockDownloader implements Contracts.P2P.Downloader {
 				height++;
 
 				const response = await this.committedBlockProcessor.process(committedBlock);
-				if (response === Contracts.Consensus.ProcessorResult.Invalid) {
+				if (response.result === Contracts.Consensus.ProcessorResult.Invalid) {
```

Comment:
We should also handle all other enum values (e.g. `ProcessorResult.Skipped` is missing)

Maybe we can just use a switch statement, the compiler should then also complain if it's non-exhaustive.

Reply:
We don't check if block status is skipped or accepted. Both values are correct. Skipped result can occur when block is accepted during the download window.